available_images.md

```diff
@@ -142,12 +142,23 @@ SageMaker Framework Graviton Containers (SM support only)
 
 NVIDIA Triton Inference Containers (SM support only)
 ============================
-**Note**:
+**Versions 23.12 and onwards**:
+1. Starting version 23.12, onwards, the sagemaker tritonserver is available in a different set of accounts than the previous ones. The new accounts are now the same as other DLCs listed on this page, making it easier to switch containers going forward. These accounts are listed in section '# Available Deep Learning Containers Images' above.
+2. Alternately, they can be obtained programmatically from the sagemaker python sdk as:
+```
+    uri = image_uris.retrieve(
+        framework=triton_framework, region=region, version=version, instance_type=instance_type
+    )
+```
+where, for example `triton_framework=sagemaker-tritonserver`, and `region=us-west-2`, `version=23.12`, `instance_type=ml.g5.12xlarge`.
+3. Unavailable versions: `24.02`.
```

Comment:
What are the available versions? Is it only 23.XX mentioned in line 158? Further down the table I see references to 22.XX and 21.08 images

Reply:
Restructured the doc a bit to clarify, versions prior to 23.12 are available in a different set of accounts, and those after are available in Asimov accounts